buidler-app/buidler.config.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
       runs: 10000,
     },
   },
+  gasReporter: {
```

Comment:
Using this configuration we can decide to run the gas report with an environment variable.